src/components/Button/BaseButton.tsx

```diff
@@ -40,6 +40,7 @@ export const BaseButton: FC<InternalButtonProps> = React.forwardRef(
             split,
             splitButtonChecked = false,
             splitButtonProps,
+            floatingButtonProps,
```

Comment:
You'll need to ensure the new `floatingButtonProps` prop is passed down through `PrimaryButton`, `SecondaryButton`, `DefaultButton`, and `NeutralButton`.

Reply:
`SplitButton` and `TwoStateButton` don't need this prop so those may be left 'as-is'.